src/functionaltests/java/com/ericsson/ei/utils/BaseRunner.java

```diff
@@ -0,0 +1,12 @@
+package com.ericsson.ei.utils;
+
+import org.junit.AfterClass;
+
+public class BaseRunner {
+
+    @AfterClass()
+    public static void shutdownAmqpBroker() {
+        String rabbitMQPort = System.getProperty("rabbitmq.port");
+        TestContextInitializer.removeBroker(Integer.parseInt(rabbitMQPort));
```

Comment:
do not do the parsing here. Create in initializer a new method removeBroker that take a string and do parsing there before calling the other method removeBroker that takes the int.